src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const transpositionRobot = {};
+  const prop = Object.keys(robot).concat(Object.values(robot));
+
+  for (let i = 0; i < prop.length; i++) {
+    for (let j = i + 1; j < prop.length; j++) {
+      if (prop[i] === prop[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
Unnecessary double loop. This solution is bad in terms of logic and performance.
Try to figure out how to solve it another way. If you need help, please ask for it in fe_chat1.